lemur/certificates/schemas.py

```diff
@@ -68,7 +68,7 @@ def default_notification(self, data):
 
 class CertificateInputSchema(CertificateCreationSchema):
     name = fields.String()
-    common_name = fields.String(required=True, validate=validators.common_name)
+    common_name = fields.String(validate=validators.common_name, missing="")
```

Comment:
The `missing=""` seems like its solving something that can't handle None? If something breaks without this, should be fixed to handle CN not present. If we do need this then a comment could be helpful.

Reply:
Current code assumes common name is always present in many places, since we had `required=True`. So I preferred to keep ` missing=""` avoid risk of not handling CN not present. I'll add comment as suggested.